plugin/mutate/source/dextool/plugin/mutate/backend/database/schema.d

```diff
@@ -548,6 +551,41 @@ struct MutationScoreHistoryTable {
     double score;
 }
 
+/** All functions that has been discovered in the source code.
+ * The `offset_begin` is where instrumentation points can be injected.
+ */
+@TableName(srcCovTable)
+@TableForeignKey("file_id", KeyRef("files(id)"), KeyParam("ON DELETE CASCADE"))
+@TableConstraint("file_offset UNIQUE (file_id, begin, end)")
+struct CoverageCodeRegionTable {
+    long id;
+
+    @ColumnName("file_id")
+    long fileId;
+
+    /// the region in the files, in bytes.
+    uint begin;
+    uint end;
```

Comment:
Hmm, so this will be all the functions for one file, but saved separately in the db.

Could it be a problem with a lot of db entries in the end?

Reply:
Nevermind, I was thinking of a solution where each row in the table is growing with every function, but that does not seem more reasonable at all.

Reply:
> Hmm, so this will be all the functions for one file, but saved separately in the db.
> 
> Could it be a problem with a lot of db entries in the end?

that is correct but the data should be pretty minimal as you see